Source/Pawnmorphs/Esoteria/Thoughts/Worker_HasMutations.cs

```diff
@@ -16,34 +19,45 @@ namespace Pawnmorph.Thoughts
     /// </summary>
     public class Worker_HasMutations : ThoughtWorker
     {
-
-        private List<Thought> _scratchList = new List<Thought>(); 
-
-
-
         /// <summary>
         /// return the thought state for the given pawn 
         /// </summary>
         /// <param name="p"></param>
         /// <returns></returns>
         protected override ThoughtState CurrentStateInternal(Pawn p)
         {
-            if (!def.IsValidFor(p)) return false;
+            if (!def.IsValidFor(p)) 
+                return ThoughtState.Inactive;
+
             MutationTracker mutTracker = p.GetMutationTracker();
-            if (mutTracker == null) return false;
```

Comment:
Returning false triggers an implicit cast to ThoughtState.Inactive, so returning that directly should be _slightly_ faster, and I think a bit more readable.